dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -65,3 +66,5 @@ def render_pdf_for_each_html_page(html_pages, html_dir, pdf_dir):
         exit_code = subprocess.call(['wkhtmltopdf', 'file://{}'.format(html_path), pdf_path])
         if exit_code > 0:
             logger.error("ERROR {} on {}".format(exit_code, html_page))
+            ok = False
```

Comment:
This is fine, although unless failure here is really common, it might just be simpler to throw. If rendering one of the docs didn't work, it's not completely clear to me that trying to do the rest is at all likely to succeed?